client/fresheyes-web/src/app/components/form/FormSection.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+
+import React, { useMemo, useState } from "react";
+import { GitHubServiceClient } from "@/proto/fresheyes_pb_service";
+import { PullRequest } from "@/proto/fresheyes_pb";
+import Link from "next/link";
+import Modal from "../modal/Modal";
+
+const FormSection = () => {
+  const client = useMemo(() => new GitHubServiceClient("http://localhost:50051"), []);
```

Comment:
Can we not initialize the GRPC in the client component? Maybe do it outside in another function in a separate file that we can call anywhere. I think that way we won't need the `useMemo`